rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -61,6 +61,56 @@ def test_exec(config, ssh_con):
 
     ip_and_port = s3cmd_reusable.get_rgw_ip_and_port(ssh_con)
 
+    if config.test_ops.get("s3cmd_get_objects", False):
+        log.info(
+            f"CEPH-83575477 - Verify s3cmd get: Bug 2174863 - [cee/sd][RGW] 's3cmd get' fails with EOF error for few objects"
+        )
```

Comment:
@anrao19 , please modify the log.info to reflect object get, we can add testcase details as a comment.